packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -79,424 +130,550 @@
+      <p>Acceptable value: any Flight icon name.</p>
     </dd>
-    <dt>
-      href
-      <code>
-        string
-      </code>
-    </dt>
+    <dt>href</dt>
     <dd>
-      <p>
-        If the interactive item is an external link, define a
-        <span class="dummy-code">href</span>; an
-        <span class="dummy-code">&lt;a&gt;</span>
-        element with the appropriate attributes will be rendered.
-      </p>
+      <p>This is the URL parameter that is passed down to the
+        <code>&lt;a&gt;</code>
+        element.</p>
     </dd>
-    <dt>
-      icon
-      <code>string</code>
-    </dt>
+    <dt>route/models/model/query/current-when/replace</dt>
     <dd>
-      <p>Acceptable value: any flight icon name. As per design, an icon
-        <em>must</em>
-        exist for critical list-items. All icons will be placed to the left of the list-item.</p>
+      <p>These are the parameters that are passed down as arguments to the
+        <code>&lt;LinkTo&gt;</code>
+        component. For more details about these parameters see the
+        <a
+          href="https://guides.emberjs.com/release/routing/linking-between-routes/#toc_the-linkto--component"
+          target="_blank"
+          rel="noopener noreferrer"
+        >Ember documentation</a>
+        or the
+        <a
+          href="https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/input?anchor=LinkTo"
+          target="_blank"
+          rel="noopener noreferrer"
+        >LinkTo component API specs</a>.</p>
     </dd>
-    <dt>
-      copyItemTitle
-      <code>string</code>
-    </dt>
+    <dt>...attributes</dt>
     <dd>
-      <p>Optional. For use on the <span class="dummy-code">copy-item</span> item type.</p>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+      <p>🚨
+        <em><strong>Important</strong>: in this specific component, the
+          <code class="dummy-code">...attributes</code>
+          are not spread on the root element of the component (an
+          <code class="dummy-code">&lt;li&gt;</code>
+          element) but on the underlying element/component (<code class="dummy-code">&lt;button&gt;</code>,
+          <code class="dummy-code">&lt;a&gt;</code>,
+          <code class="dummy-code">&lt;LinkTo&gt;</code>
+          or
+          <code class="dummy-code">&lt;LinkToExternal&gt;</code>
+          depending on the
+          <code class="dummy-code">@route/@href</code>
+          arguments).</em></p>
+    </dd>
+  </dl>
+
+  <h4 class="dummy-h4">ListItem::Separator</h4>
+  <p class="dummy-paragraph">Here is the API for the "separator" list item component (yielded as
+    <code class="dummy-code">Separator</code>):</p>
+  <dl class="dummy-component-props" aria-labelledby="listitem-separator-api">
+    <dt>...attributes</dt>
+    <dd>
+      <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+    </dd>
+  </dl>
```

Comment:
[issue] the separator does not need `...attributes`. There is nothing that users need to define for the separator other than to invoke it.

Reply:
they may want to add `data-test` attributes for testing reasons, we don't know

Reply:
Decided to leave it